NOTES.md

```diff
@@ -0,0 +1,12 @@
+Notes
+---
+
+- Reddit app will run if a plugin is missing
+- Reddit app will **not** run if a provider is missing
+- Reddit/Pylons uses `pkg_resources.WorkingSet` to load plugins/provider (see next note).
+- Plugins/providers need to be installed at the global level by invoking this:
```

Comment:
These are just general notes on findings when working with reddit, probably mostly going to be useful for the Ops team eventually.